app.py

```diff
@@ -0,0 +1,139 @@
+from flask import Flask, request, abort
+from git import Repo
+import ansible_runner
+import os
+import requests
+import json
+import configparser
+
+from datetime import datetime
+
+app = Flask(__name__)
+
+def appendTime(message):
+    output = "[" + datetime.utcnow().isoformat() + "]" + " " + message
+    return (output)
+
+def updateRepo(repoName, repoURL): #Update the local git repo
+
+    if os.path.isdir(repoName): #If the clone of the repo exists pull updates, if not clone it 
+        repo = Repo(repoName)
+        o = repo.remotes.origin
+        o.pull()
+        print(appendTime("Repo Pulled"))
```

Comment:
Any reason not to replace your use of `print` and `appendTime` with the `logging` module?

Reply:
No, the print statements were what I was using when I was initially creating and debugging the app. Was not totally sure how this would translate when put into production.